### PR TITLE
fix: enabled http when adding cookies

### PR DESCRIPTION
**Pull request by andrewtyshkovets**
develop

## Summary of issue

- [x]  http enabled in cookies

## Summary of change

- [x]  http enabled in cookies

### CODE REVIEW

**Review by melnikovae87 — APPROVED** [review]

File: src/main/java/com/softserveinc/dokazovi/util/CookieUtils.java [hunk 1]
```diff
@@ -28,7 +28,6 @@ private CookieUtils() {
 
 	public static void addCookie(HttpServletResponse response, String name, String value, int maxAge) {
 		Cookie cookie = new Cookie(name, value);
-		cookie.setSecure(true);
```
Comment (melnikovae87) [inline src/main/java/com/softserveinc/dokazovi/util/CookieUtils.java:31]: Your code has some smell)

File: build.gradle [hunk 2]
```diff
@@ -94,4 +94,4 @@ sonarqube {
 				"src/main/java/com/softserveinc/dokazovi/security/**," +
 				"src/main/java/com/softserveinc/dokazovi/config/**"
 	}
-}
```
Comment (melnikovae87) [inline build.gradle:97]: What is it?

**Review by melnikovae87 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (andrewtyshkovets) [reply build.gradle:97]: it is дужечка

File: src/main/java/com/softserveinc/dokazovi/util/CookieUtils.java (hunk 1 above)
Comment (andrewtyshkovets) [reply src/main/java/com/softserveinc/dokazovi/util/CookieUtils.java:31]: I hope it smells fine

File: build.gradle (hunk 2 above)
Comment (melnikovae87) [reply build.gradle:97]: Hahaha...

**Review by melnikovae87 — APPROVED** [review]

**Pull request merged**